launchscripts/launch.sh

```diff
@@ -4,6 +4,9 @@
 #
```

Comment:
Please remove the `-x` from the shebang on line 1 as it results in running the script via `./launch.sh` to unnecessarily debug print the commands being executed. This predated your changes but we might as well fix it.

Reply:
Done